### PR TITLE
feat(executor): Executor.from_hub now accepts jinahub scheme 🍾 

### PR DESCRIPTION
**Pull request by hanxiao**
Not going to lie, this is really one of my favorite features recently.

```python
from jina import Executor, DocumentArray, Document

exec = Executor.from_hub('jinahub://DummyHubExecutor')

da = DocumentArray([Document()])
exec.foo(da)
assert da.texts == ['hello']
```

### CODE REVIEW

**Review by JoanFM — CHANGES_REQUESTED** [review]
Can we avoid using `load_config` for this but expose the `load` that encapsulates the previous work done by the runtime before load_config?

`load_config` is already ugly enough. And `passing jinahub://` does not seem to be a config as it is right?

**Review by JoanFM — COMMENTED** [review]
Forget my previous message, I thought even docker could be used, but of course it cannot

File: jina/helper.py [hunk 1]
```diff
@@ -721,6 +721,7 @@ def kwargs2namespace(
         parser: ArgumentParser,
         warn_unknown: bool = False,
         fallback_parsers: List[ArgumentParser] = None,
+        positional_args: Tuple[str, ...] = None,
```
Comment (JoanFM) [inline jina/helper.py:724]: what is an example where this is used?

Comment (JoanFM) [reply jina/helper.py:724]:
```suggestion
        positional_args: Optional[Tuple[str, ...]] = None,
```

File: jina/helper.py [hunk 2]
```diff
@@ -721,6 +721,7 @@ def kwargs2namespace(
         parser: ArgumentParser,
         warn_unknown: bool = False,
         fallback_parsers: List[ArgumentParser] = None,
```
Comment (JoanFM) [inline jina/helper.py:723]:
```suggestion
        fallback_parsers: Optional[List[ArgumentParser]] = None,
```

File: jina/helper.py (hunk 1 above)
Comment (JoanFM) [reply jina/helper.py:724]: I would keep `positional_args` outside, the user can append them on its own. The name of the function explicitly says `kwargs2Namespace` so why would it have to take `positional_args`?

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: jina/helper.py (hunk 1 above)
Comment (hanxiao) [reply jina/helper.py:724]: because parser won't work

Comment (hanxiao) [reply jina/helper.py:724]: i would keep it as is as other wise people have to do this aganin

Comment (hanxiao) [reply jina/helper.py:724]: > what is an example where this is used?
in this pR

**Pull request merged**